app/konnect/gateway-manager/data-plane-nodes/transit-gateways.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to configure Transit Gateway
+---
+
+
+This guide will walk you through connecting your {{site.konnect_short_name}} managed Dedicated Cloud Gateways to AWS Transit Gateway, providing a secure and private channel for your API traffic.
+
+## Prerequsites 
+
+Before you start setting up the Transit Gateway with your {{site.konnect_short_name}} Dedicated Cloud Gateways, ensure that you have the following prerequisites:
+
+* A {{site.konnect_short_name}} Cloud Control Plane
+* AWS account with administrative privileges to create resources and manage peering.
+* The **AWS ID** which can be found in {{site.konnect_short_name}} by selecting the desired Network from **Gateway Manager** > **Networks**
+
+## Configure AWS Transit Gateway
```

Comment:
Maybe not for now, but since this is third-party instructions, can we find an AWS doc to link to that covers this instead of writing up our own? Just so maintenance is easier.

Reply:
This one might be a lost battle. I know Michael wanted us to maintain 3rd party instructions for key docs, and CGWs feels like it could be one.

Reply:
I thought about this overnight -- I think if we remove it, it should be a fast follow.